src/app/user/dtos/CreateUserDto.ts

```diff
@@ -12,6 +12,10 @@ export class CreateUserDto {
   @IsNotEmpty()
   password: string;
 
+  @ApiProperty()
+  @IsOptional()
+  phone: string;
```

Comment:
Is user phone optional in signup?

Reply:
this pr was a draft to add the condominium, the PR to update the phone field is here: https://github.com/RodrigoRVSN/set-api/pull/6

Reply:
But yes, i dont think we need to require the user to input the phone in sign up, makes more sense to keep optional for me

Reply:
Yes, makes sense for visitor, but for trustee and admin it's nescessary to have a phone number for communication between admin and trustee, in my vision

Reply:
i'll continue this in the update phone PR, i'll open this here

Reply:
done in [`00c3f29` (#6)](https://github.com/RodrigoRVSN/set-api/pull/6/commits/00c3f2986fccab847330f140ec898b24fdceb9e1) @gianlucabueno